content/getting-started/dolthub.md

```diff
@@ -74,30 +74,77 @@ We also acquired a full history of values for each cell in the database, which w
 
-### File Upload
+### Using DoltHub
 
-Uploading a file to DoltHub is the only way to publish data without using Dolt. Every repository, whether it contains data or not, has a button to upload a file. You'll be taken through a few steps in the upload wizard:
+There are a few ways to publish data without using Dolt.
 
-* Choose a base branch \(commits directly to `master` branch for empty repo\)
+![Create table](../.gitbook/assets/dolthub-create-table.png)
 
-  ![Choose a branch](../.gitbook/assets/choose-branch.png)
+#### File Upload
 
-* Choose a table name \(create a new table or update an existing table\)
+You can upload a file to create or update a table in your database. Click on "Upload a file" in the add item dropdown on the top right of a database page.
 
-  ![Choose table name](../.gitbook/assets/update-table2.png)
+![Upload a file dropdown](../.gitbook/assets/add-item-dropdown.png)
 
-* Upload a file and choose primary keys \(we currently support CSV, PSV, and XLSX files\)
-* View and commit changes
+You'll be taken through a few steps in the upload wizard.
 
-  ![Commit changes](../.gitbook/assets/commit-changes2.png)
+First, choose a base branch \(commits directly to the `main` branch for an empty database\)
 
-You can learn more about file upload from [our blog](https://www.dolthub.com/blog/2020-11-13-dolthub-upload-file/).
+![Choose a branch](../.gitbook/assets/upload-choose-branch.png)
 
-### Configuration
+Next, choose a table name \(create a new table or update an existing table\)
 
-To publish data on DoltHub using Dolt, the first thing to do is configure your copy of Dolt to recognize DoltHub as a remote to your repo. To do that, and then write to that repo, you need to login. Dolt provides a command for logging in that will launch a browser window, have you authenticate, and create token which your local Dolt will use to identify itself. The steps are straightforward:
+![Choose table name](../.gitbook/assets/upload-choose-table.png)
+
+Upload a file and optionally choose primary keys \(we currently support CSV, PSV, XLSX, and JSON files\)
+
+![Choose primary keys](../.gitbook/assets/upload-choose-pks.png)
+
+View and commit changes
+
+![Commit changes](../.gitbook/assets/commit-changes.png)
+
+You can learn more from our blog [here](https://www.dolthub.com/blog/2020-11-13-dolthub-upload-file/).
+
+#### Spreadsheet Editor
+
+As a part of the file upload process, we also added the option to create and edit spreadsheets directly in DoltHub. You will see this option during step 3.
+
+![Spreadsheet editor option](../.gitbook/assets/upload-step-3.png)
+
+If you chose to update an existing table, the spreadsheet editor will be populated with the table's existing data. This data is paginated and we're currently working on making it easier to navigate existing data in the spreadsheet editor.
```

Comment:
Same comment, don't mention any current pain points unless they are necessary to workaround (in which case they probably should not be documented yet). In this case I would delete the sentence after "existing data"